### DISCUSSION TITLE
How to join teams meeting as a teams user using callComposite.js

### DISCUSSION THREAD

**Discussion by Chandi-99** [Azure/communication-ui-library#5445] [Q&A]
for the past few weeks I am trying to joint the teams meeting from azure communication sdk as a teams user. Since I am using UI composite as described in this documentation. 
[documentation](https://azure.github.io/communication-ui-library/?path=/docs/composites-build-your-own-javascript-bundle--docs)
I can successfully connect to the meeting as a external user, but I am having trouble joining the teams meeting from my account.
I have created necessary token to join the meeting in azure sdk and I have test it on [](https://azure.github.io/communication-ui-library/?path=/story/composites-callcomposite-join-existing-call-as-teams-user--join-existing-call-as-teams-user) and I can successfully join the meeting.
Following is the code I have used.
```
      const callAdapter = await callComposite.loadCallComposite(
        {
          locator: { meetingLink: this.meetingURL },
          userId: {
            microsoftTeamsUserId: this.teamsUserIdentifier.RawId,
          },
          token: this.accessToken,
          options: {
            videoBackgroundOptions: {
              videoBackgroundImages: [],
              onResolveDependency: async () => {
                return {};
              },
            },
            onFetchProfile: async (userId: string, defaultProfile?: Profile): Promise<any | undefined> => {
              if (defaultProfile?.displayName) {
                return defaultProfile;
              }
              return { displayName: 'Unnamed Teams User' };
            }
          }
        },
        document.getElementById('call-container'),
        {
          options:{
            branding:{
              backgroundImage: {
                url: 'assets/media/background.png'
              }
            }
          }
        }
      );
```
      
this is the error I am getting.

loading call composite: Error: Microsoft Teams user identifier is not supported by AzureCommunicationCallAdapter. Instead use TeamsCallAdapter.

Since there is no documentation on how to join the teams meeting as a teams user using own JavaScript bundle, appreciate if anyone can help me out. I have reached to many platforms to find a solution, couldn't find the correct answer.

**Comment by mgamis-msft** [comment]
Hi @Chandi-99, thanks for bringing this to our attention. It looks like `loadCallComposite` is not implemented to handle Teams user ids. We will need to update that function to be able to do so. I will confirm with my team and let you know.

**Reply by Chandi-99** [reply]
Thank you
